mitodl/mit-open
# Updated hero page

### What are the relevant tickets?
- https://github.com/mitodl/hq/issues/4375

### Description (What does it do?)
Updates the hero section

### Screenshots (if appropriate):

![Screenshot 2024-05-29 at 3 39 16 PM](https://github.com/mitodl/mit-open/assets/9010790/cb5b6d62-a1b5-4f26-975c-2a999c8e2f14)

![Screenshot 2024-05-29 at 3 39 23 PM](https://github.com/mitodl/mit-open/assets/9010790/607bb9a7-88fd-418e-b218-8f2d563e785c)

https://github.com/mitodl/mit-open/assets/9010790/b1015d51-7c72-4da9-b9d0-f58757fb04eb


### How can this be tested?
1. View the homepage. Its hero section should look like [designs](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=3852-52752&m=dev)
2. Click the links. 
    - "trending" links should go to appropriate search page
    - browse by topics => topics parge
    - explore all => search page  

### Additional Context
